polaris/plugin/server_connector/server_connector.cpp

```diff
@@ -553,6 +553,12 @@ void GrpcServerConnector::TimingServerSwitch(GrpcServerConnector* server_connect
 
 ReturnCode GrpcServerConnector::SelectInstance(const ServiceKey& service_key, uint32_t timeout,
                                                Instance** instance, bool ignore_half_open) {
+  // 支持standalone模式，如果service_name为空，则直接返回埋点服务实例
+  if (service_key.name_.length() <= 0) {
```

Comment:
"service_key.name_.length() <= 0" switch to "service_key.name_.empty()",  could be better?

Reply:
> "service_key.name_.length() <= 0" switch to "service_key.name_.empty()", could be better?

fixed~